Binder build fails on master

**Describe the issue**
A clear and concise description of what the issue is.
```
/home/jovyan/.npm-global/bin/yarn -> /home/jovyan/.npm-global/lib/node_modules/yarn/bin/yarn.js
/home/jovyan/.npm-global/bin/yarnpkg -> /home/jovyan/.npm-global/lib/node_modules/yarn/bin/yarn.js
+ yarn@1.22.10
added 1 package in 0.821s
yarn lerna run build --stream
yarn run v1.22.10
error Command "lerna" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Makefile:107: recipe for target 'build-ui' failed
make: *** [build-ui] Error 1
Removing intermediate container 2c6d13661136
The command '/bin/sh -c ./binder/postBuild' returned a non-zero code: 2
```

**To Reproduce**
Steps to reproduce the behavior:
1. Open https://mybinder.org/v2/gh/elyra-ai/elyra/master?urlpath=lab/tree/binder-demo

--- a/Makefile
+++ b/Makefile
@@ -104,7 +104,7 @@ yarn-install:
 	yarn install
 
 build-ui: # Build packages
-	yarn lerna run build --stream
+	yarn && lerna run build --stream
 
 build-server: # Build backend
 	python setup.py bdist_wheel sdist

--- a/binder/postBuild
+++ b/binder/postBuild
@@ -26,7 +26,7 @@ npm config set prefix ~/.npm-global
 export PATH=$(pwd)/.npm-global/bin/:$PATH
 
 # Download and install Yarn Package Manager using npm
-npm install -g yarn
+npm install -g yarn lerna
 
 # Build Elyra Distribution
 make release

